aiomisc_pytest/pytest_plugin.py

```diff
@@ -27,6 +29,182 @@
     from inspect import isasyncgenfunction
 
 
+ProxyProcessorType = t.Callable[[bytes], t.Union[bytes, t.Awaitable[bytes]]]
+
+
+class TCPProxyClient:
+    __slots__ = (
+        "client_reader", "client_writer",
+        "server_reader", "server_writer",
+        "chunk_size", "tasks", "loop",
+        "delay", "closing",
+        "read_processor", "write_processor",
+    )
+
+    def __init__(
+        self, client_reader: asyncio.StreamReader,
+        client_writer: asyncio.StreamWriter,
+        chunk_size: int = 64 * 1024,
+    ):
+
+        self.loop = asyncio.get_event_loop()
+        self.client_reader = client_reader  # type: asyncio.StreamReader
+        self.client_writer = client_writer  # type: asyncio.StreamWriter
+
+        self.server_reader = None  # type: t.Optional[asyncio.StreamReader]
+        self.server_writer = None  # type: t.Optional[asyncio.StreamWriter]
+
+        self.tasks = ()                     # type: t.Iterable[asyncio.Task]
+        self.chunk_size = chunk_size        # type: int
```

Comment:
I'd rather remove comments indention aligning. It's not convenient to realign after changes.